__device-tests__/gutenberg-editor-unsupported-block-visual.test.js

```diff
@@ -42,28 +31,18 @@ describe( 'Gutenberg Editor Visual test for Unsupported Block', () => {
 		const unsupportedBlock = await editorPage.getBlockAtPosition(
 			editorPage.blockNames.unsupported
 		);
-		await unsupportedBlock.click();
-		// Wait for the block to be selected
-		await editorPage.driver.pause( 1000 );
 
 		// Visual test check
-		let screenshot = await takeScreenshot();
-		expect( screenshot ).toMatchImageSnapshot();
-
-		// Select title to unfocus the block
-		const titleElement = await editorPage.getTitleElement();
-		await titleElement.click();
-
-		await editorPage.dismissKeyboard();
-
-		// Visual test check
-		screenshot = await takeScreenshot();
+		const screenshot = await takeScreenshotByElement( unsupportedBlock, {
+			padding: 7,
+		} );
 		expect( screenshot ).toMatchImageSnapshot();
 
 		await toggleDarkMode( editorPage.driver, false );
 	} );
 
-	it( 'should be able to open the unsupported block web view editor', async () => {
+	// Disabled temporarily
+	it.skip( 'should be able to open the unsupported block web view editor', async () => {
```

Comment:
I'll work on enabling this once I update the E2E testing functionality to be able to pass the `capabilities` option in a separate PR.